packages/drivekit_trip_analysis/flutter_drivekit_trip_analysis_ios/ios/.swiftlint.yml

```diff
@@ -0,0 +1,24 @@
+excluded:
+  - Classes/IOSCoreApi.swift # Generated file
```

Comment:
I think `IOSTripAnalysisApi.swift` is missing

(also for core `.swiftlint.yml` file)

Reply:
good catch

Reply:
there is a config file by project. The  core .swiftlint.yml file should have this line only